.gitignore

```diff
@@ -0,0 +1,4 @@
+/_build
```

Comment:
#### Ignoring `/_build`

I actually considered doing this, but I wanted to make it easier for myself and others to use the precompiled components on systems that don't have Elixir.  My main concern was to be able to run this from my NAS that has an easy way to install Erlang but not Elixir.  I presume, though, that once I run `mix escript.build`, then all the required bits are packaged inside the executable, which I can download directly from Github onto my NAS (?) and therefore should probably exclude the `/_build` directory.

I guess if I ever want to use just part of an app, it is far better to import that functionality into the source code, rather than trying to work with precompiled beam files...

#### Including test coverage analysis in repository

I included this as a way of trying to be helpful to others quickly scanning the repository - hoping that the fact that I care about tests and test coverage would be seen as a positive... ??